client/src/Components/Flows/TippingFlow/SelectAmount/SelectAmount.js

```diff
@@ -6,17 +6,17 @@ import { bindActionCreators } from 'redux';
 import { setSelectedTipAmount } from '../../../../Actions';
 
 const SelectAmount = (props) => {
-  
   const [amount, setAmount] = useState(5)
+  const { user } = props.UserReducer;
 
   const submitHandler = () => {
     props.setSelectedTipAmount(amount)
-    props.history.push('/tip')
+    props.history.push('/tip/select-receiver')
   }
 
   // TO-DO: Make into separate Utility Function with semi-exhaustive currency list
```

Comment:
I'm thinking maybe we should simplify the functionality to handle just one currency initially. What do you think?

Reply:
I agree! Would massively simplify the development process. I think we should opt for USD.

Will add to the tip-related routes card on Trello